dist/Data-Dumper/t/qr.t

```diff
@@ -16,9 +19,15 @@ use Data::Dumper;
     my $q = q| \/ |;
     use Data::Dumper;
     my $qr = qr{$q};
-    eval Dumper $qr;
+    eval add_my_to_dump( Dumper $qr );
     ok(!$@, "Dumping $qr with XS") or diag $@, Dumper $qr;
     local $Data::Dumper::Useperl = 1;
-    eval Dumper $qr;
+    eval add_my_to_dump( Dumper $qr );
     ok(!$@, "Dumping $qr with PP") or diag $@, Dumper $qr;
 }
+
+sub add_my_to_dump {
```

Comment:
This makes the test pass, but is it really the right thing to do?

It means that any code that assumes `eval Dumper $foo` will work will break, and I think there's quite a bit of such code out there.